composite-aeson-refined/composite-aeson-refined.cabal

```diff
@@ -1,13 +1,13 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.33.0.
+-- This file has been generated from package.yaml by hpack version 0.34.2.
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: f4a9dd8025ca20a24dd7b1c000231332627bcb754951b64c68ac5744e908b8cc
+-- hash: cddbe2a4ad5a7be16b1cf3ab1f9288a70091002fbcacb4e9e5aabc4e89ac1cd1
 
 name:           composite-aeson-refined
-version:        0.7.2.0
+version:        0.7.3.0
```

Comment:
I'll end up redoing these changes anyway as 0.7.3.0 has already been released, so I would say don't worry about version bumping in the future as it's part of the release process. That said, thanks for the reminder about the ChangeLog